e2e/cypress/plugins/external_request.js

```diff
@@ -36,7 +37,7 @@ module.exports = async ({baseUrl, user, method = 'get', path, data = {}}) => {
         });
     } catch (error) {
         if (error.response) {
-            response = error.response;
+            response = error.message;
```

Comment:
Could you check again the changes here?  For `error.response`, I think that's correct but I could be wrong.